app/Views/Templates/components/forms/button.blade.php

```diff
@@ -29,6 +29,9 @@
         case 'link':
             $typeClass = 'btn-link text-primary hover:bg-primary/20 ';
             break;
+        case 'transparent':
```

Comment:
Why? Isn't this the ghost type? Or link?

Reply:
fixed it